.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+profile=black
```

Comment:
What is this setting for? It looks like it might be an editor config setting that could have accidentally been checked in.

Reply:
No, this is a linter that needs this configuration. I cannot remember what it is responsible for, but I believe it is for sorting includes for various programs